apps/studio/src/server/modules/page/page.router.ts

```diff
@@ -100,6 +110,79 @@ export const pageRouter = router({
         ...siteMeta,
       }
     }),
+
+  reorderBlock: protectedProcedure
+    .input(reorderBlobSchema)
+    .mutation(async ({ input: { pageId, from, to, blocks, siteId }, ctx }) => {
+      // NOTE: we have to check against the page's content that we retrieve from db
+      // we adopt a strict check such that we allow the update iff the checksum is the same
+      const fullPage = await getFullPageById({ resourceId: pageId, siteId })
+
+      if (!fullPage?.content) {
+        // TODO: we should probably ping on call
+        throw new TRPCError({
+          code: "NOT_FOUND",
+          message:
+            "Unable to load content for the requested page, please contact Isomer Support",
+        })
+      }
+
+      const actualBlocks = fullPage.content.content
+
+      if (!isEqual(blocks, actualBlocks)) {
+        throw new TRPCError({
+          code: "CONFLICT",
+          message:
+            "Someone on your team has changed this page, refresh the page and try again",
```

Comment:
This page has been recently updated. Please refresh the page to fetch the latest changes and try again.

Reply:
this has been cleared with @sehyunidaaa already, so idt we should change

Reply:
ah ok good point, i like your suggestion! with minor lingo tweaks, maybe can do

> "This page was recently updated. Refresh the page to get the latest changes."

@seaerchin